socorro/external/fs/crashstorage.py

```diff
@@ -2,12 +2,9 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-import datetime
+import gzip
 import json
```

Comment:
I'm curious about the order of imports for within the same type of import. Is there a standard within the project? It seems that a lot of it alphabetical.

Reply:
Yeah!

So, we do groups:

* python stdlib
* 3rd party libraries
* django things
* socorro things

In each group, we order them alphabetically.

We use flake8-isort to lint it. Though it looks like we have it shut off for everything except the e2e-tests directory.

https://github.com/timothycrosley/isort